src/style.css

```diff
@@ -0,0 +1,126 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 600;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 20px 7.6px 7.5px ;
+}
```

Comment:
try to don't reset the margin without setting the padding, I think the result should be the same

Reply:
I removed the margin in "ul". But when I remove the margin in the body, the result immediately changes to the wrong one.